src/main/java/frc/team2412/robot/Subsystems/IntakeOnOffSubsystem.java

```diff
@@ -0,0 +1,70 @@
+package frc.team2412.robot.Subsystems;
+
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.Subsystems.constants.IntakeConstants;
+import frc.team2412.robot.Subsystems.constants.IntakeConstants.IntakeLastMotor;
+
+public class IntakeOnOffSubsystem extends SubsystemBase {
+
+	private CANSparkMax m_intakeFrontMotor;
+	private CANSparkMax m_intakeBackMotor;
+	private SpeedControllerGroup m_intakes = new SpeedControllerGroup(m_intakeFrontMotor, m_intakeBackMotor);

```

Comment:
this will error out, initialize m_intakes after the motors are set in the constructor